### ISSUE TITLE
Assignment 3 Review

### ISSUE THREAD

**Issue by raghu2094** [airavata-courses/CookieArmy#95]
Hi Team,

I am starting with the Assignment 3 Review.
Add my public key to the master node for review.
Ping me on slack and I will share my public key.

Regards,
Raghottam

**Comment by raghu2094** [comment]
Hey team

Did you try load testing by increasing your services. In WIKI you haven't mentioned your system's capacity limit.

Regards,
Raghottam

**Comment by jkumar10** [comment]
Yes, we have tried load testing on individual services as well as on the system. Our system was able to take an average thread dump of around 1200 threads without breaking. Currently we have tried load testing on 3 instances of each micro-service.
https://github.com/airavata-courses/CookieArmy/wiki/JMeter-Testing

Scripts:
https://github.com/airavata-courses/CookieArmy/tree/TestOps
